testsuite/features/profiles/cloud_aws/Docker/authprofile/sles15sp4.repo

```diff
@@ -0,0 +1,6 @@
+[sles15sp4]
+name=sles15sp4
+enabled=1
+autorefresh=0
+baseurl=http://download.suse.de/ibs/SUSE/Products/SLE-Module-Basesystem/15-SP4/x86_64/product/
```

Comment:
Can we access to that IBS repo from an AWS instance? 🤔

Reply:
Hmm, good catch. We need to configure the mirror